packages/cypress-angular-dev-server/src/lib/create-angular-webpack-config.ts

```diff
@@ -26,28 +27,31 @@ function getCompilerConfig(wco: WebpackConfigOptions) {
 }
 
 export async function createAngularWebpackConfig(config: {
+  workspaceRoot: string;
   projectRoot: string;
   sourceRoot: string;
   tsConfig: string;
   buildOptions?: Record<string, unknown>;
 }): Promise<Configuration> {
   const projectRoot = normalize(config.projectRoot);
-  const workspaceRoot = projectRoot;
   const sourceRoot = normalize(config.sourceRoot);
+  const workspaceRoot = normalize(config.workspaceRoot ?? '');
   const tsConfig = normalize(config.tsConfig);
 
   const buildOptions = config.buildOptions ?? {};
   const polyfills = (buildOptions.polyfills as string) ?? 'placeholder';
   const styles = (buildOptions.styles as ExtraEntryPoint[]) ?? [];
   const scripts = (buildOptions.scripts as ExtraEntryPoint[]) ?? [];
   const assets = (buildOptions.assets as AssetPattern[]) ?? [];
+  const stylePreprocessorOptions =
+    buildOptions?.stylePreprocessorOptions as StylePreprocessorOptions;
   const main = (buildOptions.main as string) ?? null;
   const index = (buildOptions.index as IndexUnion) ?? null;
 
   const normalizedOptions = normalizeBrowserSchema(
-    workspaceRoot,
     projectRoot,
-    sourceRoot,
+    projectRoot,
+    workspaceRoot ?? sourceRoot,
```

Comment:
What about the following: 

```suggestion
    workspaceRoot,
    projectRoot,
    sourceRoot,
```

Reply:
The last one needs to be workspaceRoot instead of sourceRoot for the included style paths to work.